src/components/CollectionFilters/LineFilter.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import * as FiltersActions from '../../actions/filters';
+import * as FilterSetsActions from '../../actions/filterSets';
+
+class LineFilter extends Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  filterIsApplied() {
+    const filters = this.props.filters;
+    for (let i = 0; i < filters.length; i++) {
+      if (filters[i].slug === this.props.filter.slug) {
+        return i;
+      }
+    }
+    return -1;
+  }
+
+  handleClick(event) {
```

Comment:
It'd be cool to make a parent class at some point to hold some of this shared logic. And then each one of these could extend the parent.

Reply:
Agree. For now, waiting to see what about these filters is unique once I have them appropriately querying ES. My sense is that they're actually the same thing, but holding off on combining them until I know that.